packages/cli/test/unit/util/deploy/create-git-meta.test.ts

```diff
@@ -125,4 +126,28 @@ describe('createGitMeta', () => {
       await fs.rename(join(directory, '.git'), join(directory, 'git'));
     }
   });
+  it('fails when .git is corrupt)', async () => {
+    const directory = fixture('git-corrupt');
+    try {
+      await fs.rename(join(directory, 'git'), join(directory, '.git'));
+
+      const data = await createGitMeta(directory, client.output);
+
+      // `getLastCommit()` runs `git log -1`, which searches parent directories as well.
```

Comment:
How about if we copy `directory` to a temp directory i.e. `path.join(os.tmpdir(), 'git-corrupt')`, so that the fixture is "outside" of `vercel/vercel`.